File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        vector<bool> result;
        

        int mx = candies[0];
        for(int i=0;i<candies.size();i++)
        {
            if(candies[i]>=mx)
                mx = candies[i];
        }
        for(int i=0; i<candies.size(); i++)
        {
            if(candies[i] + extraCandies >= mx)
                result.push_back(true);
            else
                result.push_back(false);
            
        
        }
       return result; 
    }
};

File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        result = []
        for i in range(len(candies)):
            result.append(0)
        c = 0
        for i in candies:
            if i + extraCandies >= max(candies):
                result[c] = True
            else:
                result[c] = False
            c += 1
        return result
                
            
        